data/income.csv

```diff
@@ -0,0 +1,521 @@
+,name,group,pct,total,order,region,id
+0,Alabama,<10000,10.2,1837292,1,south,1

```

Comment:
The last column is the state id, right? Should we move it right after the state column?